pgtype/interval.go

```diff
@@ -132,29 +132,25 @@ func (encodePlanIntervalCodecText) Encode(value any, buf []byte) (newBuf []byte,
 
 	if interval.Days != 0 {
 		buf = append(buf, strconv.FormatInt(int64(interval.Days), 10)...)
-		buf = append(buf, " day"...)
+		buf = append(buf, " day "...)
```

Comment:
If months is 0, this will produce " day hh:mm:ss"

Reply:
nope, the number goes just before. There is a test with 0 months that looks good: https://github.com/jackc/pgx/pull/2035/files#diff-deeaa7f4bfb6897c6bb770fb12921f2350ce60ca3a2a5416c84a1ec23bb2b11eR151

Reply:
Ah, my mistake!